phpstan.neon

```diff
@@ -82,5 +82,16 @@ parameters:
       - '#PHPDoc tag @var for variable \$router contains unknown class Laravel\\Lumen\\Routing\\Router\.#'
       - '#Call to method addRoute\(\) on an unknown class Laravel\\Lumen\\Routing\\Router\.#'
 
+  # Recent Mockery versions do not account for the last arg possibly being a closure
+  - path: tests/Unit/Subscriptions/Iterators/AuthenticatingSyncIteratorTest.php
+    message: '#Parameter \#2 \.\.\.\$args of static method Mockery::mock\(\) expects array\|class-string<Nuwave\\Lighthouse\\Subscriptions\\SubscriptionGuard>\|Nuwave\\Lighthouse\\Subscriptions\\SubscriptionGuard, Closure given\.#'
 
+  - path: tests/Integration/Schema/Directives/BuilderDirectiveTest.php
+    message: '#Return type of call to static method Mockery::mock\(\) contains unresolvable type\.#'
+
+  # Laravel added generics to Arr::first in 11.x, not sure how to handle them for now.
+  - path: src/OrderBy/OrderByDirective.php
+    message: '#Unable to resolve the template type TValue in call to method static method Illuminate\\Support\\Arr::first\(\)#'
+  - path: src/Testing/TestResponseMixin.php
+    message: '#Unable to resolve the template type TValue in call to method static method Illuminate\\Support\\Arr::first\(\)#'
```

Comment:
This one is a PITA to debug. I think it's fine to silence for now if tests pass. Let me know if you're good with it.